drivers/hmis/app/controllers/hmis/app_settings_controller.rb

```diff
@@ -11,7 +11,8 @@ class Hmis::AppSettingsController < Hmis::BaseController
   def show
     okta_enabled = ENV['HMIS_OKTA_CLIENT_ID'].present? && ENV['OKTA_DOMAIN'].present?
 
-    logo = ENV['LOGO']
+    hmis_logo = ENV['HMIS_LOGO']
+    logo = hmis_logo && !hmis_logo.blank? ? hmis_logo : ENV['LOGO'] # prefer HMIS_LOGO if provided, otherwise LOGO
```

Comment:
Hmm I think this could be made more succinct. `presence` will convert `'' => nil` which is a bit neater:
```
# prefer HMIS_LOGO if provided, otherwise LOGO (which is shared with the warehouse)
logo = ENV['HMIS_LOGO']&.presence || ENV['LOGO']&.presence
if logo
  ...
```